ee/server/service/mdm_profiles.go

```diff
@@ -0,0 +1,78 @@
+package service
+
+import "text/template"
+
+type fileVaultProfileOptions struct {
+	PayloadIdentifier    string
+	Base64DerCertificate string
+}
+
+var fileVaultProfileTemplate = template.Must(template.New("").Option("missingkey=error").Parse(`<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should we make the UUIDs template arguments instead of hard-coding them?

Reply:
It won't make a difference! UUIDs need to be unique per device. We're doing the same for the enrollment profile